src/modules/audio/audio.h

```diff
@@ -54,9 +59,12 @@ float lovrSourceGetVolume(Source* source);
 void lovrSourceSetVolume(Source* source, float volume);
 bool lovrSourceGetSpatial(Source *source);
 void lovrSourceSetPose(Source *source, float position[4], float orientation[4]);
+void lovrSourceGetPose(Source *source, float position[4], float orientation[4]);
 uint32_t lovrSourceGetTime(Source* source);
 void lovrSourceSetTime(Source* source, uint32_t sample);
 struct SoundData* lovrSourceGetSoundData(Source* source);
+const char *lovrSourceGetSpatializerName();
+intptr_t *lovrSourceGetSpatializerMemoField(Source *source);
```

Comment:
Couldn't come up with a better way to do this (you need to be able to both get and set the memo field rapidly and frequently).